facebook/delphiFacebook/R/variables.R

```diff
@@ -40,11 +60,14 @@ is_selected <- function(vec, selection) {
         # selection ("") or missing (NA) as missing, for generality.
         NA
       } else {
-        selection %in% resp
+        any(resp == selection)
```

Comment:
I see you've replaced `%in%` in two spots; is this known to be faster for some reason?

Reply:
I think `%in%` calls `match`:

```r
> `%in%`
function (x, table) 
match(x, table, nomatch = 0L) > 0L
```

Perhaps the extra bookkeeping done by `match` to find the index of the first occurrence? Dunno